native_client/deepspeech.h

```diff
@@ -15,6 +15,9 @@ struct ModelState;
 
 struct StreamingState;
 
+struct Metadata;
```

Comment:
`Metadata` and `MetadataItem` are supposed to be manipulated by the user, contrary to `StreamingState`, so I don't think they should just be opaque structure.